docs/en/operations/table_engines/mergetree.md

```diff
@@ -45,7 +45,10 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 [SETTINGS name=value, ...]
 ```
 
-For a description of request parameters, see [request description](../../query_language/create.md).
+For a description of parameters, see the [CREATE query description](../../query_language/create.md).
+
+!!!note "Note"
```

Comment:
* lost space after !!! ?
* needs link/mention of feature flag to enable it

Reply:
There is the [Data Skipping Indexes](https://clickhouse.yandex/docs/en/operations/table_engines/mergetree/#data-skipping-indices-experimental) section. I think its enough.